cpp/change-notes/2021-08-24-implicit-downcast-from-bitfield.md

```diff
@@ -0,0 +1,3 @@
+lgtm,codescanning
+* Extend query `ImplicitDowncastFromBitfield` to detect implicit
+  downcast on references
```

Comment:
```suggestion
* The query `cpp/implicit-bitfield-downcast` now accounts for C++ reference types, which leads to move true positive results.
```

I think this is how the old Semmle docs team has taught us to write change notes. Notably, queries are referred to by id (or name and id), and TPs are called "true positive results".

Cc @geoffw0 for a second pair of eyes.

Reply:
Typo: move
which leads to **more** true positive results.

Reply:
Fixed, thanks! Clearly a second pair of eyes was needed :smile:.

Reply:
Yes, we refer to queries by query ID: `cpp/implicit-bitfield-downcast`

Or sometimes with the query name and ID: "Implicit downcast from bitfield" (`cpp/implicit-bitfield-downcast`)

But not by file name in formal places such as this.  Query IDs have the advantages of being very easy to search for, and in theory they should never change and there should never be two distinct queries with the same ID (which can happen with query names, e.g. the equivalent query for two different target languages).

Other than that, as with all documentation try to use simple English, avoid idioms and avoid words with apostrophes ("doesn't").  Technical words are generally fine though.  What you have now LGTM.